generator/namespaces.py

```diff
@@ -329,10 +330,14 @@ def namespaces(custom_namespaces, generated_sql_uri, app_listings_uri, disallowl
             _merge_namespaces(namespaces, custom_namespaces)
 
     disallowed_namespaces = yaml.safe_load(disallowlist.read()) or {}
+    disallowed_regex = [
+        fnmatch.translate(namespace) for namespace in disallowed_namespaces
+    ]
+    disallowed_namespaces_pattern = re.compile("|".join(disallowed_regex))
 
     updated_namespaces = {}
     for namespace, _ in namespaces.items():
-        if namespace not in disallowed_namespaces:
+        if not disallowed_namespaces_pattern.match(namespace):
```

Comment:
this needs to use fullmatch to ensure it's matching the entire string:
```suggestion
        if not disallowed_namespaces_pattern.fullmatch(namespace):
```

note: `re.Pattern.match` "Matches zero or more characters at the beginning of the string", and `fnmatch.translate` will suffix each pattern with `\Z` (end of input), so this isn't required, but it's still safer to use `fullmatch` in case someone changes how `disallowed_namespaces_pattern` is generated.